WordPress/Classes/Models/Blog.m

```diff
@@ -259,6 +259,18 @@ - (NSArray *)sortedPostFormatNames
     }];
 }
 
+- (NSArray *)sortedConnections
+{
+    NSSortDescriptor *sortServiceDescriptor = [[NSSortDescriptor alloc] initWithKey:@"service"
+                                                                          ascending:YES
+                                                                           selector:@selector(caseInsensitiveCompare:)];
+    NSSortDescriptor *sortExternalNameDescriptor = [[NSSortDescriptor alloc] initWithKey:@"externalName"
+                                                                               ascending:YES
+                                                                                selector:@selector(caseInsensitiveCompare:)];
```

Comment:
Good point. I don't think we're translating service names, so it makes sense to only localize account names 👍